st2common/st2common/util/pack_management.py

```diff
@@ -339,8 +339,8 @@ def apply_pack_owner_group(pack_path):
 
         if exit_code != 0:
             # Non fatal, but we still log it
-            LOG.debug('Failed to change owner group on directory "%s" to "%s": %s' %
-                      (pack_path, pack_group, stderr))
+            LOG.debug('Failed to change owner group on directory "{}" to "{}":'.format(pack_path, pack_group))
```

Comment:
probably best to leave the logging format the same, in this case keeping `stderr` on the same line instead of another debug line.